src/Sylius/Bundle/CoreBundle/Resources/config/services/context.xml

```diff
@@ -42,7 +42,7 @@
             <argument type="service" id="sylius.context.customer" />
             <argument type="service" id="sylius.context.channel" />
             <argument type="service" id="sylius.repository.order" />
-            <tag name="sylius.context.cart" priority="-555" />
+            <tag name="sylius.context.cart" priority="-888" />
```

Comment:
It should probably be mentioned in the UPGRADE file, isn't it?

Reply:
This might also be affected: https://docs.sylius.com/en/latest/book/orders/cart-flow.html